package.json

```diff
@@ -8,7 +8,8 @@
     "del": "^2.2.2",
     "govuk_frontend_toolkit": "5.0.3",
     "gulp": "3.8.7",
-    "gulp-jasmine-phantom": "2.0.1"
+    "gulp-jasmine-phantom": "2.0.1",
+    "jasmine-core": "2.6.4"
```

Comment:
No, it doesn't. If it's closed, the added line (102) in toggleOptionSelect triggers the update when it's actually closed.